src/test/app/forms/form.ts

```diff
@@ -105,6 +105,19 @@ describe('Form', () => {
       expect(form.valueFor('address[city]')).to.equal('London')
     })
 
+    it('should return raw value if value associated with given field fails validation', () => {
```

Comment:
The `form.rawDataFor` function has nothing to do with validation errors. Form might have no validation error and outcome will be the same. Please correct the test body and description to remove the confusion.